reliability-v2/utils/openshift-qe-workers-infra-workload-commands.sh

```diff
@@ -0,0 +1,771 @@
+#           OPENSHIFT_WORKLOAD_NODE_INSTANCE_MEMORYSIZE=64Gi
+#-------------------------------------------------------------------------------------
+
+#IF_INSTALL_INFRA_WORKLOAD=true/false
+if test ! -f "${KUBECONFIG}"
+then
+	echo "No kubeconfig, can not continue."
+	exit 0
+fi
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+SHARED_DIR=${SHARED_DIR:=""}
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1090
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+IF_INSTALL_INFRA_WORKLOAD=${IF_INSTALL_INFRA_WORKLOAD:=true}
+if [[ ${IF_INSTALL_INFRA_WORKLOAD} != "true" ]];then
+   echo "No need to install infra and workload for this OCP cluster"
+   exit 0
+fi
+
+# Download jq
+jq --version
+if [ $? != 0 ]; then
+  if [ ! -d /tmp/bin ];then
+    mkdir /tmp/bin
+    export PATH=$PATH:/tmp/bin
+    curl -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /tmp/bin/jq
+    chmod ug+x /tmp/bin/jq
+  fi
+fi
+
+#Get Basic Infrastructue Architecture Info
+node_arch=$(oc get nodes -ojsonpath='{.items[*].status.nodeInfo.architecture}')
+platform_type=$(oc get infrastructure cluster -ojsonpath='{.status.platformStatus.type}')
+platform_type=$(echo $platform_type | tr -s 'A-Z' 'a-z')
+node_arch=$(echo $node_arch | tr -s " " "\n"| sort -u)
+all_machinesets=$(oc -n openshift-machine-api get machineset -ojsonpath='{.items[*].metadata.name}{"\n"}')
+machineset_list=$(echo $all_machinesets | tr -s ' ' '\n'| sort -u| grep -v -i -E "infra|workload|win|rhel"| head -n3)
+machineset_count=$(echo $all_machinesets | tr -s ' ' '\n'| sort -u| grep -v -i -E "infra|workload|win|rhel"| head -n3 |wc -l)
+total_worker_nodes=$(oc get nodes -l node-role.kubernetes.io/worker= -oname|wc -l)
+
+scale_type=""
+#Currently only support AWS reference to ROSA settings
+if [[ $total_worker_nodes -ge 100 ]];then
+	scale_type=medium
+elif [[ $total_worker_nodes -ge 25 && $total_worker_nodes -lt 100 ]];then
+	scale_type=small
+elif [[ $total_worker_nodes -ge 1 && $total_worker_nodes -lt 25 ]];then
+	scale_type=extrasmall
+fi
+
+######################################################################################
+#             CHANGE BELOW VARIABLE IF YOU WANT TO SET DIFFERENT VALUE               #
+######################################################################################
+SET_ENV_BY_PLATFORM=${SET_ENV_BY_PLATFORM:=$platform_type}
+echo SET_ENV_BY_PLATFORM is $SET_ENV_BY_PLATFORM
+case ${SET_ENV_BY_PLATFORM} in
+	aws)
+     #ARM64 Architecture:
+	   if [[ $node_arch == "arm64" ]];then
+	      if [[ ${scale_type} == "medium" ]];then
+                OPENSHIFT_INFRA_NODE_INSTANCE_TYPE=m6g.12xlarge
```

Comment:
the arm instance size still not correct, please download latest code from openshift/release.

Reply:
@qiliRedHat It seems the instance type of arm64 still not correct, The latest one https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/workers-infra-workload/openshift-qe-workers-infra-workload-commands.sh#L606

Reply:
@liqcui I updated it, thanks!